blog.py

```diff
@@ -316,13 +319,21 @@ def _load_post(self, post):
             content,
             empty,
             0,
+            retitle,
         ])
     
     def _is_ess_empty(self, lines: List[str]) -> int:
         content = ''.join(line.strip() for line in lines)
         is_empty = re.sub(r'\*\*.+\*\*', '', content).replace('**', '') == ''
         return 1 if is_empty else 0
 
+    def _is_rename_title(self, path_, title_) -> int:
```

Comment:
similarly, confusing function name, `_is_name_title_mismatch` or `_name_title_mismatch`

Reply:
changed to _is_name_title_mismatch